src/libraries/Directory.Build.targets

```diff
@@ -240,6 +237,22 @@
     <ExcludeFromPackage Condition="'$(TargetFramework)' == '$(NetFrameworkCurrent)' and '$(ExcludeCurrentFullFrameworkFromPackage)' == 'true'">true</ExcludeFromPackage>
   </PropertyGroup>
 
+  <!-- Adds System.Runtime.Versioning*Platform* annotation attributes to < .NET 5 builds -->
+  <Choose>
+    <When Condition="$(TargetFramework.StartsWith('netstandard')) or $(TargetFramework.StartsWith('netcoreapp')) or '$(TargetFrameworkIdentifier)' == '.NETFramework'">
```

Comment:
Nit: use `TargetFrameworkIdentifier` for `netstandard` as well

Reply:
I would remove the nit, as the TFI really should be used here.

Reply:
Would it be better to the individual projects to include this?

 Unlikely nullable attributes that are needed everywhere, this is only really needed in a few projects.

Reply:
Yeah I agree. This seems better to be included individually by projects. We would avoid issues like: https://github.com/dotnet/runtime/pull/39265#discussion_r455264167

Reply:
I guess the only downside is the duplication of the condition. Maybe we should leave it here, but we can add a property in the project that states to include this file and do the tfm conditions here?

Reply:
> Would it be better for the individual projects to include this?

>  we should leave it here, but we can add a property in the project that states to include this file and do the tfm conditions here

That's a very good idea. I've introduced a new property `<IncludePlatformAttributes>true</IncludePlatformAttributes>` and extended the condition